deploy/clowdenv.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ephemeral-clowdenvironment
+
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  # Custom Resource defined as part of the Clowder API
+  kind: ClowdEnvironment
+  metadata:
+    name: ${ENV_NAME}
+  spec:
+    targetNamespace: ${NAMESPACE}
+
+    # Providers our app to consume configuration
+    # data automatically based on our request
+    providers:
+
+      # provides a k8s service on port 8000
+      web:
+        port: 8000
+        privatePort: 8080
+        mode: operator
+
+      # provides a prometheus metrics port on 9000
+      metrics:
+        port: 9000
+        mode: operator
+        path: "/metrics"
+
+      # Creates a kafka pod in the targetNamespace
+      kafka:
+        namespace: default
+        clusterName: crc-cluster
+        mode: local
```

Comment:
I suggest to remove Kafka for now, deployments will be faster I guess. But feel free to keep it.